packages/core/src/Select/MultiSelect.tsx

```diff
@@ -99,10 +99,26 @@ export function MultiSelect<V extends string = string>({
           e.stopPropagation()
           onChange?.(value.filter(v2 => v2 !== v))
         },
+        handleKeyDown: (e: React.KeyboardEvent) => {
```

Comment:
This should probably be moved to inside the chip component.

Reply:
Moved it. 

I think the `ChipProps` `onRemove` should be renamed to `onClickRemove`. Or even better keep `onRemove` but remove the mouse event so line `packages/core/src/Chip/index.tsx:45` will be `readonly onRemove?: () => void` as I think the click event should be handled inside the component and not by the one using a Chip. Just as we handle the key event now.

But it would be a breaking change.

Reply:
Breaking changes are fine. Sounds reasonable!